### ISSUE TITLE
RDP error while connectiong to Windows Server <2022

### ISSUE THREAD

**Issue by mbzik** [electerm/electerm#3585]
### Electerm Version and download file extension(Electerm版本和下载文件后缀)

electerm-1.39.119-win-x64-installer.exe

### Platform detail (平台详情)

Microsoft Windows NT 10.0.19045.0 x64

### What steps will reproduce the bug?(重新问题的详细步骤)

Trying to connect via RDP to Windows Server 2008 - 2019. (lower than 2022)
![image](https://github.com/user-attachments/assets/75c28301-5cdb-44e9-8db5-17842b1cbcf9)
After connectiong i get grey screen and message in logs. 

While trying to investigate, realized that the node-rdpjs library can cause SSL issues. But I have not been able to figure out why this error occurs. Maybe there is a way to skip certificate validation when connecting?

### What should have happened?(期望的结果)

Establishing the connection

**Comment by zxdong262** [comment]
https://github.com/electerm/electerm/wiki/RDP-limitation

**Comment by mbzik** [comment]
> https://github.com/electerm/electerm/wiki/RDP-limitation

Yes, this fixes the Windows Server 2022 connection error, but with older versions of Windows Server (tested with Windows Server 2016, 2019) the connection does not work.

**Comment by zxdong262** [comment]
> > https://github.com/electerm/electerm/wiki/RDP-limitation
> 
> Yes, this fixes the Windows Server 2022 connection error, but with older versions of Windows Server (tested with Windows Server 2016, 2019) the connection does not work.

Seems limited by the rdpjs lib, currently no other rdp lib to use

**Comment by mbzik** [comment]
> > > https://github.com/electerm/electerm/wiki/RDP-limitation
> > 
> > 
> > Yes, this fixes the Windows Server 2022 connection error, but with older versions of Windows Server (tested with Windows Server 2016, 2019) the connection does not work.
> 
> Seems limited by the rdpjs lib, currently no other rdp lib to use

Is there way to try this one - node-freerdp2 - https://www.npmjs.com/package/node-freerdp2?
It's have an option `certIgnore: true`

**Comment by mbzik** [comment]
Is there an option to switch to this library? I was able to replace the backend part, but was unable to load the screen on the frontend.

**Comment by zxdong262** [comment]
> Is there an option to switch to this library? I was able to replace the backend part, but was unable to load the screen on the frontend.

Should be able to fix in next release https://github.com/electerm/electerm/commit/648afaf27a1b5f5af5ed875c3c5ad6ebb8744154

Let's see how it goes

**Comment by mbzik** [comment]
Is it on 1.40.6 update?
In logs i got an error: 
rdp error Error: 2238464:error:1000012e:SSL routines:OPENSSL_internal:KEY_USAGE_BIT_INCORRECT:..\..\third_party\boringssl\src\ssl\ssl_cert.cc:605:

Is it any certIgnore analogs in new library?

**Comment by zxdong262** [comment]
> Is it on 1.40.6 update? In logs i got an error: rdp error Error: 2238464:error:1000012e:SSL routines:OPENSSL_internal:KEY_USAGE_BIT_INCORRECT:....\third_party\boringssl\src\ssl\ssl_cert.cc:605:
> 
> Is it any certIgnore analogs in new library?

As I tested, works in win11, got another user report not working in win10, I do not know reason yet

**Comment by paja6** [comment]
I can see the same error when connecting from Windows 11 to Windows Server 2016 and 2019. I have not tested other Windows Server versions. I tested it with both NLA disabled and enabled.

**Comment by zxdong262** [comment]
Got someone told me that it is related to additional key
![1remote](https://github.com/user-attachments/assets/23f7714c-c239-41b1-9670-d306e1bebd82)

**Comment by paja6** [comment]
Is the screenshot from the Credentials Manager in Windows? If yes, I do not see such a record on the client or the server.

**Comment by zxdong262** [comment]
-> https://github.com/electerm/electerm/issues/3764
